### ISSUE TITLE
Remove the "I Don't Like It" content reporting option (or make it actually do something)

### ISSUE THREAD

**Issue by dariusk** [hometown-fork/hometown#1202]
### Pitch

There is an "I don't like it" option when you report a post in Mastodon 3.5.5 and in v4. It currently doesn't do anything, it says "well then you should unfollow, block, or mute this person!" This option should either be removed, or made to report something with an "I don't like it" category.

### Motivation

It's bad to offer a false choice to a user under the report menu.

**Comment by Gaelan** [comment]
I believe Glitch has a patch to remove it you could steal.

**Comment by dariusk** [comment]
EDIT: The below does not seem to be working for most people even though it is somehow working on my server. Weird.

In the meantime admins can add this custom css via the admin -> site settings -> custom CSS panel

```
.report-dialog-modal__container > div > label.dialog-option:nth-child(1) {
  display: none;
}
```

**Comment by hazel-bohon** [comment]
We have added the above CSS into our custom CSS panel but it doesn't seem to be working

**Comment by dariusk** [comment]
@Nix-Bohon I may have had to restart mastodon services for it to work, and perhaps to recompile assets as well. I can test this tomorrow when I'm back at a computer.

**Comment by rabbithearth** [comment]
> We have added the above CSS into our custom CSS panel but it doesn't seem to be working

As a follow up to this, other custom CSS added in the same way since Nix's comment seems to be working, and we can see this edit in custom.css. I also tried adding !important to it in case it needed to override some other CSS, but that doesn't seem to help, either.

**Comment by dariusk** [comment]
@rabbithearth Sorry about that. I added a note that it probably won't work as a temporary fix.

**Comment by dariusk** [comment]
Okay, ultimately my solution here is to keep the "I don't like it" but have it send you to the same options you get when you choose "Other". So you can still report it and you can provide your reasoning. I think that on small servers especially, "I find this person annoying" can be a reason for an admin to block someone on behalf of everyone.

**Comment by dariusk** [comment]
This is fixed in the upcoming v1.0.8 release.
